src/js/subtraction/components/List.jsx

```diff
@@ -1,58 +1,62 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { connect } from "react-redux";
 import { checkAdminRoleOrPermission } from "../../administration/utils";
 import { Badge, LegacyScrollList, LoadingPlaceholder, NoneFoundBox, ViewHeader, ViewHeaderTitle } from "../../base";
 import { findSubtractions } from "../actions";
 import SubtractionItem from "./Item";
 import SubtractionToolbar from "./Toolbar";
 
-export class SubtractionList extends React.Component {
-    componentDidMount() {
-        if (!this.props.fetched) {
-            this.props.onLoadNextPage(this.props.term, 1);
+export const SubtractionList = ({ onLoadNextPage, documents, page, page_count, total_count, fetched, term }) => {
+    useEffect(() => {
+        if (!fetched) {
+            onLoadNextPage(term, 1);
         }
-    }
+    }, []);
 
-    renderRow = index => <SubtractionItem key={index} index={index} />;
+    const renderRow = index => <SubtractionItem key={index} index={index} />;
```

Comment:
This is best defined outside the component to prevent redefinition on every render